### ISSUE TITLE
Reservation section

### ISSUE THREAD

**Issue by MarcoBarber** [gabzon/experiensa#16]
Add start date and end date

**Comment by MarcoBarber** [comment]
question: is the left agenda supposed to show the availability? how will it be updated automatically?

**Comment by valenciafg** [comment]
Now the administrator user can configure the date that are not available to be booked. This can be done by accessing the Administration menu appareance menu, then options and settings hotels

Previously in the Info tab should be selected that the website will be used for hotels

After this, the booking calendar will be updated and disable the unavailable dates

**Comment by MarcoBarber** [comment]
So I understand the admin will update availability manually: is my understanding correct?

**Comment by valenciafg** [comment]
Yes, the administrator can load ranges of dates that the hotel does not have availability. For example: from 10/16/04 to 10/16/10, also from 1/11/16 to 11/25/16

**Comment by gabzon** [comment]
@MarcoBarber please make a test, if it works close the issue
